quickstarts/microsoft.databricks/databricks-all-in-one-template-for-vnet-injection-privateendpoint/azuredeploy.parameters.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
```

Comment:
remove the param values from here and use defaultValues in the template

Reply:
Thanks, @bmoore-msft for the quick review.

Reply:
This still needs to be done